presto-docs/src/main/sphinx/release/release-339.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Release 339 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Add :func:`approx_most_frequent`. (:issue:`3425`)
```

Comment:
Put add below fix

Reply:
I put the new features and additions first per offline conversations.